<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExpansionHeader</class>
 <widget class="QDialog" name="ExpansionHeader">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>214</width>
    <height>131</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Expansion Header</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0" alignment="Qt::AlignRight">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Mode</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="modeBox"/>
   </item>
   <item row="1" column="0" alignment="Qt::AlignRight">
    <widget class="QLabel" name="Param1Label">
     <property name="text">
      <string>Drive time, ms</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="Param1">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0" alignment="Qt::AlignRight">
    <widget class="QLabel" name="Param2Label">
     <property name="text">
      <string>Cooldown time, ms</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="Param2">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ExpansionHeader</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ExpansionHeader</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
